projects/angular-auth-oidc-client/src/lib/extractors/jwk.extractor.spec.ts

```diff
@@ -71,7 +71,7 @@ describe('JwkExtractor', () => {
     n: 'wq0vJv4Xl2xSQTN75_N4JeFHlHH80PytypJqyNrhWIp1P9Ur4-5QSiS8BI8PYSh0dQy4NMoj9YMRcyge3y81uCCwxouePiAGc0xPy6QkAOiinvV3KJEMtbppicOvZEzMXb3EqRM-9Twxbp2hhBAPSAhyL79Rwy4JuIQ6imaqL0NIEGv8_BOe_twMPOLGTJhepDO6kDs6O0qlLgPRHQVuKAz3afVby0C2myDLpo5YaI66arU9VXXGQtIp8MhBY9KbsGaYskejSWhSBOcwdtYMEo5rXWGGVnrHiSqq8mm-sVXLQBe5xPFBs4IQ_Gz4nspr05LEEbsHSwFyGq5D77XPxGUPDCq5ZVvON0yBizaHcJ-KA0Lw6uXtOH9-YyVGuaBynkrQEo3pP2iy1uWt-TiQPb8PMsCAdWZP-6R0QKHtjds9HmjIkgFTJSTIeETjNck_bB4ud79gZT-INikjPFTTeyQYk2jqxEJanVe3k0i_1vpskRpknJ7F2vTL45LAQkjWvczjWmHxGA5D4-1msuylXpY8Y4WxnUq6dRTEN29IRVCil9Mfp6JMsquFGTvJO0-Ffl0_suMZZl3uXNt23E9vGreByalWHivYmfpIor5Q5JaFKekRVV-U1KDBaeQQaHp_VqliUKImdUE9-GXNOIaBMjRvfy0nxsRe_q_dD6jc_GU',
     e: 'AQAB',
   } as JsonWebKey;
-  let keys: JsonWebKey[] = [key1, key2, key3, key4, key5, key6, key7, key8, key9];
+  const keys: JsonWebKey[] = [key1, key2, key3, key4, key5, key6, key7, key8, key9];
```

Comment:
These tests fail local with the error:

```
  An error was thrown in afterAll
  Uncaught TypeError: Cannot read properties of undefined (reading 'buildErrorName')
  TypeError: Cannot read properties of undefined (reading 'buildErrorName')
```

While building/running the app this error is gone 🤔

Reply:
It seems like TS changed the behavior of static members?
As a work around I moved the static members to const variables.